audonnx/core/testing.py

```diff
@@ -89,20 +100,86 @@ def create_model(
 
     # create model
 
-    model = audonnx.Model(
-        graph,
+    model = Model(
+        object,
         transform=transform,
+        device=device,
     )
 
     return model
 
 
-def _identity_graph(
+def create_onnx_object(
         shapes: typing.Sequence[typing.Sequence[int]],
-        dtype: int,
-        opset_version: int,
+        *,
+        dtype: int = onnx.TensorProto.FLOAT,
+        opset_version: int = 14,
 ) -> onnx.ModelProto:
-    r"""Create identity graph."""
+    r"""Create test ONNX object.
```

Comment:
In the documentation of `audonnx.Model` we call it "ONNX model object" which let me first think this here is something more generic, but I don't think it is. I would propose to change to
```suggestion
    r"""Create test ONNX model object.
```
and maybe rename the function to `create_model_object()` or `create_model_proto()`?

Reply:
I didn't want to call it `create_model_object` since we already have `create_model`, which also returns an `object`. So let's got with `create_model_proto()` then.

Reply:
Another solution would be call it `create_onnx()` and just remove the `object` part as both functions create objects.
But I think I would prefer `create_model_proto()` as you suggested.

Reply:
Ok, renamed accordingly (see description). I also changed to:

![image](https://user-images.githubusercontent.com/10383417/174802022-53bd4263-ea70-4047-bd9f-296e3298769f.png)

Reply:
Cool, thanks.